contracts/true-currencies/tokens/TrueUSDNoDelegate.sol

```diff
@@ -0,0 +1,1292 @@
+/**
+ *Submitted for verification at Etherscan.io on 2020-09-25
+*/
+
+/**
+ *  :::==== :::====  :::  === :::===== :::  === :::===  :::====
+ *  :::==== :::  === :::  === :::      :::  === :::     :::  ===
+ *    ===   =======  ===  === ======   ===  ===  =====  ===  ===
+ *    ===   === ===  ===  === ===      ===  ===     === ===  ===
+ *    ===   ===  ===  ======  ========  ======  ======  =======
+ */
+
+pragma solidity 0.6.10;
```

Comment:
Unfortunately, no we can't fix it in this PR.

We must merge this PR into `main` first to preserve this exact file and its compiler settings in the `main` commit log, and then afterward fix lint and other issues in follow-up PR(s) in a way that we can diff directly on `main`.

The biggest problem we're trying to address is that any changes to this file will result in a contract that does not compile exactly to the bytecode that's already on-chain at the address in PR #1171. And if we don't have a record of what generates the on-chain bytecode for addresses in deployments.json, then the deployment build is not reproducible. This has caused immense headaches for our contracts in the past, where we've tried to untangle woven commit histories and not been able to figure out exactly what we had deployed, because the commit hash at which we modified `deployments.json` did not generate the exact bytecode deployed to the addresses.

The unreproducible build issue is also why this inelegant flattened `TrueUSDNoDelegate.sol` exists in the first place. We could not trust the code in our repo, since it does not compile exactly to what was deployed on-chain and we couldn't find a previous match. So we had to create a workaround whereby we downloaded our own source code from Etherscan, verified that it compiled correctly, modified it into this `TrueUSDNoDelegate.sol` hack, and deployed this flattened file back on-chain.